src/ssl.c

```diff
@@ -24561,13 +24561,52 @@ int  wolfSSL_sk_num(WOLF_STACK_OF(WOLFSSL_ASN1_OBJECT)* sk)
 
 void* wolfSSL_sk_value(WOLF_STACK_OF(WOLFSSL_ASN1_OBJECT)* sk, int i)
```

Comment:
With making the function generic to all stack types the signature of (WOLFSSL_STACK* sk, int i) may be better.

Reply:
Also updated wolfSSL_sk_num